### PR TITLE
Fix TypeError on column widths specified as percentages

### PR DESCRIPTION
**Pull request by JanEgner**
### Short description
Fix recently introduced mistake in column with handling

### Resolved issues

Fixes: #731

### CODE REVIEW

File: xhtml2pdf/xhtml2pdf_reportlab.py [hunk 1]
```diff
@@ -827,9 +827,9 @@ def wrap(self, availWidth, availHeight):
         # IMPORTANT!!! We can not substitute the private value
         # self._colWidths therefore we have to modify list in place
         for i, colWidth in enumerate(newColWidths):
-            if (colWidth is not None) or (colWidth == "*"):
```
Comment (viniciusd) [inline xhtml2pdf/xhtml2pdf_reportlab.py:830]: This second part isn't really an issue, but it is a truism as the first branch will be true already

**Review by timobrembeck — APPROVED** [review]
Sorry for messing this up and thank you for the quick fix :muscle:

**Comment by viniciusd** [comment]
@timobrembeck sorry for talking about releases again, but do you anticipate a new release with this fix? Maybe with #730 as well

**Comment by timobrembeck** [comment]
@viniciusd I've released version `0.2.13` now, sorry for the inconvenience.

**Pull request merged**